docs/index.md

```diff
@@ -62,6 +62,65 @@ connection "hcloud" {
 
 - `token` - API token from Hetzner Cloud.
 
+## Multi-Account Connections
+
+You may create multiple hcloud connections:
+
+```hcl
+connection "hcloud_dev" {
+  plugin = "hcloud"
+  token  = "RCgLZcAGBBGqkr8lTFXCfLhCjNNY6OJtzhg4ZsDRdMvmUwAeLssvLWyRRRdV8lkB"
+}
+
+connection "hcloud_qa" {
+  plugin = "hcloud"
+  token  = "RCgLZcAGBBGqkr8lTFXCfLhCjNNY6OJtzhg4ZsDRdMvmUwAeLzzrLWyRRRdV8lkB"
+}
+
+connection "hcloud_prod" {
+  plugin = "hcloud"
+  token  = "RCgLZcAGBBGqkr8lTFXCfLhCjNNY6OJtzhg4ZsDRdMvmUwAeLmmyLWyRRRdV8lkB"
+}
+```
+
+Each connection is implemented as a distinct [Postgres schema](https://www.postgresql.org/docs/current/ddl-schemas.html). As such, you can use qualified table names to query a specific connection:
+
+```sql
+select * from hcloud_qa.hcloud_volume
+```
+
+You can multi-account connections by using an [**aggregator** connection](https://steampipe.io/docs/using-steampipe/managing-connections#using-aggregators). Aggregators allow you to query data from multiple connections for a plugin as if they are a single connection.
+
+```hcl
+connection "hcloud_all" {
+  plugin      = "hcloud"
+  type        = "aggregator"
+  connections = ["hcloud_dev", "hcloud_qa", "hcloud_prod"]
+}
+```
+
+Querying tables from this connection will return results from the `hcloud_dev`, `hcloud_qa`, and `hcloud_prod` connections:
+
+```sql
+select * from hcloud_all.hcloud_volume
+```
+
+Alternatively, can use an unqualified name and it will be resolved according to the [Search Path](https://steampipe.io/docs/guides/search-path). It's a good idea to name your aggregator first alphbetically, so that it is the first connection in the search path (i.e. `hcloud_all` comes before `hcloud_dev`):
```

Comment:
```suggestion
Alternatively, you can use an unqualified name and it will be resolved according to the [Search Path](https://steampipe.io/docs/guides/search-path). It's a good idea to name your aggregator first alphabetically so that it is the first connection in the search path (i.e. `hcloud_all` comes before `hcloud_dev`):
```